### PR TITLE
Sets up CODEOWNERS file

### PR DESCRIPTION
**Pull request by mitchhentges**
Set up according to [this RFC](https://github.com/mozilla-releng/releng-rfcs/blob/master/rfcs/0002-repo-automation-code-owners.md).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# You can also refer to a user by an email address that has been added
+# to their GitHub account, for example user@example.com.
+# https://help.github.com/articles/about-codeowners/
+
+
+# By default the Android Components team will be the owner for everything in
+# the repo. Unless a later match takes precedence.
+* @mozilla-mobile/act
+/automation @mozilla-mobile/releng
+/CODEOWNERS @mozilla-mobile/releng
```
Comment (pocmo) [inline .github/CODEOWNERS]: Can ACT own CODEOWNERS too?

Comment (pocmo) [inline .github/CODEOWNERS]: Can ACT own CODEOWNERS too?

**Comment by mitchhentges** [comment]
Missed the main `CODEOWNERS`, thanks for catching that :)
Since ACT owns all (`*`), I don't believe that they need to be manually set as code owner

**Comment by pocmo** [comment]
For code reviews though: 

> Code owners are automatically requested
> for review when someone opens a pull request that modifies code that
> they own. Order is important; the last matching pattern takes the most
> precedence.

**Pull request merged**